src/schema/v2/partner/createPartnerContactMutation.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+
+// Check whats required and whats optional
+interface Input {
+  partnerID: string
+  name?: string
+  position?: string
+  canContact?: boolean
+  email?: string
+  emailConfirmation?: string
+  phone?: string
+  locationID?: string
+}
+
+const CreatePartnerContactMutationResponseType = new GraphQLObjectType<
```

Comment:
This should be a union type, using the `...OrError` pattern. Check out this one (just a recent artwork import mutation) as a model - https://github.com/artsy/metaphysics/blob/db820b25e11c901a52c2ad0a771b4151fbceea32/src/schema/v2/ArtworkImport/cancelArtworkImportMutation.ts

Reply:
Thanks for the breadcrumbs 🙏🏻

Reply:
Adjusted!